quay-enterprise/insert-custom-cert.md

```diff
@@ -0,0 +1,50 @@
+# Adding TLS Certificates to the Quay Enterprise Container
+
+To add custom TLS certificates to Quay Enterprise, create a new directory named `extra_ca_certs/` beneath the Quay Enterprise config directory. Copy any required site-specific TLS certificates to this new directory.
+
+## Example
+
+### View certificate to be added to the container: 
+
+```
+$ cat storage.crt  
+-----BEGIN CERTIFICATE-----
+MIIDTTCCAjWgAwIBAgIJAMVr9ngjJhzbMA0GCSqGSIb3DQEBCwUAMD0xCzAJBgNV
+[...]
+-----END CERTIFICATE-----
+```
+
+### Create certs directory and copy certificate there:
+
+```
+$ mkdir quay/config/extra_ca_certs
+
+$ cp storage.crt quay/config/extra_ca_certs/
+
+$ ls quay/config/extra_ca_certs/
```

Comment:
we should use `tree` instead of `ls` to show the `config.yaml` inside the `quay/config` in order to clarify what directory `quay/config` is